cms/static/sass/course-unit-mfe-iframe-bundle.scss

```diff
@@ -3,7 +3,7 @@
 @import 'neat/neat'; // lib - Neat
 @import 'vendor/bi-app/bi-app-ltr'; // set the layout for left to right languages
 @import 'build-v1'; // shared app style assets/rendering
-@import 'cms/theme/variables-v1';
+@import './partials/cms/theme/_variables-v1.scss';
```

Comment:
[question] I'm not sure that we need to change this import. What is the reason of this change?

Reply:
old path is not really correct, and not all changes are overrides by it